feature-account-api/src/main/java/io/novafoundation/nova/feature_account_api/domain/model/MetaAccount.kt

```diff
@@ -60,7 +60,7 @@ fun LightMetaAccount(
 class MetaAccount(
     override val id: Long,
     val chainAccounts: Map<ChainId, ChainAccount>,
-    val proxies: List<ProxyAccount>,
+    val proxy: ProxyAccount?,
```

Comment:
Ideally this should only be present for PROXIED wallet type but I see that we cant easily change LightMetaAccount.Type to match that